app.js

```diff
@@ -2,7 +2,10 @@
 require('dotenv').config()
 
 const path = require('path')
+const lru = require('lru')
```

Comment:
https://www.npmjs.com/package/tiny-lru should be faster.

Reply:
tiny-lru results were slower

```
Running 30s test @ http://localhost:9080/rest/api/flights
10 connections

Stat         Avg     Stdev  Max     
Latency (ms) 3.15    1.46   86.82   
Req/Sec      2759.54 275.88 3037    
Bytes/Sec    9.51 MB 984 kB 10.5 MB 

83k requests in 30s, 286 MB read
null { title: 'acmeair-flights',
  url: 'http://localhost:9080/rest/api/flights',
  socketPath: undefined,
  requests: 
   { average: 2759.54,
     mean: 2759.54,
     stddev: 275.88,
     min: 1729,
     max: 3037,
     total: 82769,
     sent: 82779 },
  latency: 
   { average: 3.15,
     mean: 3.15,
     stddev: 1.46,
     min: 0,
     max: 86.826824,
     p50: 3,
     p75: 3,
     p90: 5,
     p99: 8,
     p999: 12,
     p9999: 40,
     p99999: 86 },
  throughput: 
   { average: 9507089.07,
     mean: 9507089.07,
     stddev: 984194.7,
     min: 5980611,
     max: 10504983,
     total: 286297971 },
  errors: 0,
  timeouts: 0,
  duration: 30,
  start: 2018-06-29T15:02:41.938Z,
  finish: 2018-06-29T15:03:11.964Z,
  connections: 10,
  pipelining: 1,
  non2xx: 0,
  '1xx': 0,
  '2xx': 82769,
  '3xx': 0,
  '4xx': 0,
  '5xx': 0 }
```